vms/platformvm/block/builder/builder_test.go

```diff
@@ -320,27 +315,26 @@ func TestGetNextStakerToReward(t *testing.T) {
 	}
 }
 
-func TestBuildBlock(t *testing.T) {
```

Comment:
I deleted the original `TestBuildBlock` since the mocks are tricky to get right now that we verify them first. Re-adding proper tests here: https://github.com/ava-labs/avalanchego/pull/2466